app/(dashboard)/edit/[id]/page.tsx

```diff
@@ -13,6 +14,20 @@ const EditCvPage = async ({ params: { id } }: EditCvPageProps) => {
   const titles = (await fetchTitles()) || [];
   const cv = await fetchCv(id);
   const initialUserSkills = cv?.cv_skill || [];
+  const supabase = createServerComponentClient();
+
+  const {
+    data: { session },
+  } = await supabase.auth.getSession();
+
+  const userId = session?.user.id;
+
+  const { data } = await supabase
+    .from("admins")
+    .select("id")
+    .filter("user_id", "eq", userId);
```

Comment:
[If I understood correctly](https://supabase.com/docs/reference/javascript/filter#:~:text=This%20is%20an%20escape%20hatch%20%2D%20you%0Ashould%20use%20the%20specific%20filter%20methods%20wherever%20possible.), this should be 
```suggestion
    .eq("user_id", userId);
```

Reply:
agree